pydantic/functional_serializers.py

```diff
@@ -15,11 +15,30 @@
 
 @slots_dataclass(frozen=True)
 class PlainSerializer:
+    """
+    A plain serializer class.
+
+    Attributes:
+        func (core_schema.SerializerFunction): The serializer function.
+        json_return_type (core_schema.JsonReturnTypes | None, optional): The JSON return type. Defaults to `None`.
+        when_used (Literal['always', 'unless-none', 'json', 'json-unless-none'], optional): The serialization condition.
+            Defaults to 'always'.
+    """
+
     func: core_schema.SerializerFunction
     json_return_type: core_schema.JsonReturnTypes | None = None
     when_used: Literal['always', 'unless-none', 'json', 'json-unless-none'] = 'always'
 
     def __get_pydantic_core_schema__(self, source_type: Any, handler: GetCoreSchemaHandler) -> core_schema.CoreSchema:
+        """Gets the Pydantic core schema.
```

Comment:
Is the `"""Get` intentional? It seems like usually the `"""` go on a line by itself. Not sure if there's something with the formatting I'm missing here, I do generally prefer to have the actual body of the docstring start on the following line though. (Similar to the other docstrings in this file.)